aws/internal/test/testing.go

```diff
@@ -0,0 +1,114 @@
+package test
```

Comment:
It would be super awesome if these new testing functions had unit testing so they can be updated without needing to verify with various acceptance tests. 👍  I'm also not the biggest fan of the package being named `test` since the function names already relay that (I'd suggest `tfawsresource`), but that's a personal nit since its internal to this module until it is migrated to the SDK.

Reply:
no problem I can change the package name